app/controllers/application_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  helper_method :cart
+  helper_method :cart, :current_user, :current_merchant?, :current_admin?
```

Comment:
We figured out what helper_method is for--it allows methods in ApplicationController to be called from within a view.